django/contrib/admin/tests.py

```diff
@@ -97,14 +99,29 @@ def wait_until_invisible(self, css_selector, timeout=10):
             timeout
         )
 
-    def wait_page_loaded(self):
+    def wait_page_ready(self, timeout=10):
+        """
+        Block until page has started to load.
```

Comment:
This docstring is the same as for `wait_page_loaded()`. Do we even need it in either case? Or perhaps the docstring should explain the difference between these two methods.

Reply:
Yes, good point. I gave it a shot, be I fear it could still be improved.